contrib/cirrus/build.sh

```diff
@@ -16,7 +16,7 @@ then
     in_podman --rm $IN_PODMAN_NAME $0
 else
     echo "Compiling buildah (\$GOSRC=$GOSRC)"
-    showrun make clean ${CROSS_TARGET:-all} ${CROSS_TARGET:+CGO_ENABLED=0}
+    showrun make clean ${CROSS_TARGET:-all} ${CROSS_TARGET:+CGO_ENABLED=0 cross}
```

Comment:
Note: This is fugly.  Followup beautification for the cross-compile test is here: https://github.com/containers/buildah/pull/2886